15_mini_project_work/3_magic_ball_game.py

```diff
@@ -8,11 +8,30 @@
 USER_NAME_PROMPT = 'Введи свое имя:\n'
 EXCLAMATION_SIGN = '!'
 USER_NAME_GREETING = 'Добро пожаловать, '
+CONTINUE_GAME_PROMPT = 'Хочешь задать еще один вопрос?\nНажми "1", затем ENTER,' \
+                       ' если ДА\nНажми любую клавишу, затем ENTER, если НЕТ\n'
+ENTER_QUESTION_PROMPT = 'Введи свой вопрос:\n'
+YES_RESPONSE = '1'
+FAREWELL_MESSAGE = 'Возвращайся если возникнут вопросы!'
+
+
+def game_run():
+    game_is_on = True
+    while game_is_on:
+        question = input(ENTER_QUESTION_PROMPT).lower()
+        answer = choice(ANSWERS)
+        print(answer)
+
+        continue_game = input(CONTINUE_GAME_PROMPT)
+        if continue_game != YES_RESPONSE:
```

Comment:
Try to guess yourself - what is wrong here

Reply:
I tried. For this step's requirement I don't see any problem so far.

Reply:
I tried. For this step's requirement I don't see any problem so far.